web/src/app/schedules/temp-sched/TempSchedAddShiftForm.tsx

```diff
@@ -55,12 +55,12 @@ export default function TempSchedAddShiftForm(): JSX.Element {
             name='end'
             hint={
               <Typography
+                component='a'
                 className={classes.typography}
                 variant='caption'
-                color='textSecondary'
                 onClick={() => setManualEntry(false)}
               >
-                Configure as duration?
+                Configure as duration
               </Typography>
```

Comment:
The linter complained for a reason, now that we know it's an accessibility issue, let's fix this properly.